config.yaml

```diff
@@ -15,4 +15,9 @@ options:
     type: string
     description: >
       External hostname this gunicorn app should respond to (required).
-    default: 'foo.internal'
+    default: "foo.internal"
+  external_port:
+    type: int
+    description: >
```

Comment:
Why splitting key and value here?

Reply:
We expect the port to be a number, just to be more consistent with types. All the other config options are typed as well.

Reply:
I meant the description from its value, I would have expected to see:
`description: External port for Ingress configuration.` instead of splitting that in two lines.